playbooks/roles/backups/tasks/main.yml

```diff
@@ -87,6 +96,27 @@
   when: BACKUPS_MONGO and "{{ item.value }}" != ""
   tags: ['backups']
 
+- name: Configure backup settings for Mongo Backups Monitor
+  lineinfile: >
+    dest=/etc/cron.d/edx_backup_monitor_mongo
+    regexp="^{{ item.key }}="
+    line="{{ item.key }}={{ item.value }}"
+    state=present
+    insertbefore=BOF
+  with_dict:
+    BACKUP_DIR: '{{ BACKUPS_DIR }}'
+    BACKUP_BUCKET: '{{ BACKUPS_BUCKET }}'
+    BACKUP_USER: '{{ BACKUPS_MONGO_USER }}'
+    BACKUP_PASSWORD: '{{ BACKUPS_MONGO_PASSWORD }}'
```

Comment:
You shouldn't need `BACKUP_DIR`, `BACKUP_USER`, or `BACKUP_PASSWORD` for this cron job. Those are used to auth against Mongo, then dump the backups to the VM's filesystem. Are you seeing errors when those values aren't included?

Reply:
didn't need them. just duplicated the stanza from the other one. will remove.